extension/src/LanguageFunctions.ts

```diff
@@ -296,6 +296,7 @@ export async function __refreshXlfFilesFromGXlf(gXlfFilePath: vscode.Uri, langFi
             if (!isNullOrUndefined(langTransUnit)) {
                 if (!langTransUnit.hasTargets()) {
                     langTransUnit.targets.push(getNewTarget(langIsSameAsGXlf, gTransUnit));
+                    langIsSameAsGXlf ? langTransUnit.insertCustomNote('New translation. Target copied from source.') : langTransUnit.insertCustomNote('New translation');
```

Comment:
I think these text should be gathered in an object so they'll be easier to keep track off :)

Reply:
or just another Enum?